src/dbnode/namespace/types.go

```diff
@@ -357,3 +365,21 @@ type DownsampleOptions struct {
 	// (e.g. rollup/recording rules).
 	All bool
 }
+
+// Status is the status of the namespace.
+type Status uint8
+
+const (
+	Unknown Status = iota
```

Comment:
nit: will also need a comment, also typically this would be `UnknownStagingStatus` or something along those lines just for more context for intellisense